saleor/graphql/checkout/mutations.py

```diff
@@ -113,10 +113,11 @@ def update_checkout_shipping_method_if_invalid(
         checkout.save(update_fields=["shipping_method", "last_change"])
 
 
-def check_lines_quantity(variants, quantities, country, channel_slug):
+def check_lines_quantity(variants, quantities, country, channel_slug, new_line=True):
```

Comment:
The flag parameter is a bad idea, but if we must use it here, I would rename it to something like `allow_zero_quantity=False`. In the docstring, I would add: "By default, zero quantity is not allowed, but if this validation is used for updating existing checkout lines, `allow_zero_quantities` can be set to `True` and checkout lines with this quantity can be later removed."